docs/cosmicrays.rst

```diff
@@ -110,6 +110,12 @@ This will update the file in place, as the flag is set:
    Example of cosmic-ray flagging from convolution from a Laplacian kernel and bilinear interpolation to highlight the differences.
 
       
+AstroDrizzle Cosmic Ray flagging
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Slitlessutils provides functionality to utilize AstroDrizzle as part of Cosmic Ray handling.
+Slitlessutils only stores the final rectified mosaics and the original files (updated in place) by default.
```

Comment:
The docs should describe exactly what is changed (in place) in the original files (e.g., DQ arrays are updated with bit = X for identified cosmic rays, etc.)

Reply:
This part I do not feel qualified to write. I'd appreciate it if you or @Russell-Ryan could provide me with the verbiage that should be included here. Thank you!